packages/perseus/src/widgets/interactive-graphs/interactive-graph-question-builder.ts

```diff
@@ -315,20 +441,25 @@ class SegmentGraphConfig implements InteractiveFigureConfig {
     }
 }
 
-class LinearConfig implements InteractiveFigureConfig {
+class LinearGraphConfig implements InteractiveFigureConfig {
     private startCoords?: CollinearTuple;
+    private correctCoords: CollinearTuple;
 
-    constructor(startCoords?: CollinearTuple) {
-        this.startCoords = startCoords;
+    constructor(options?: {
+        coords?: CollinearTuple;
+        startCoords?: CollinearTuple;
+    }) {
+        this.startCoords = options?.startCoords;
+        this.correctCoords = options?.coords ?? [
+            [-5, 5],
```

Comment:
Note: These are the default correct coords (before we were using an arbitrary set of coords)